js/panel-manager.js

```diff
@@ -0,0 +1,57 @@
+// var count = 0;
+
+var PanelManager = {
+//$(document).ready(function(){
+//  $('#btn-add').click(function(e){
+  init: function(e){
+    $('.panel-manager').load('templates/panel-manager.html', function(e){
+      $('#slidebar').on('click', function(e){
+        $('#slider').toggle('slide', { direction: 'right' }, 300);
+        $('#slidebar').animate({
+          'right' : $('#slidebar').css('right') == '0px' ? '249px' : '0px'
+        }, 300);
+        setTimeout(function(){
+          $('#icon-button').toggleClass('glyphicon-chevron-right glyphicon-chevron-left');
+        }, 300);
+      });
+    });
+  },
+
+  addPanel: function(view){
+//     count++;
+    var count = $('.sideways').children().length;
+//    e.preventDefault();
```

Comment:
Please cleanup the commented code a bit. I'd recommend using bulk comments /*...*/ and add "TODO(liana): ..." to describe why the commented code is left. If the code will no longer be needed, then remove the commented code.